config

```diff
@@ -26,7 +31,7 @@ encrypt_media=1
 ###############################################################################
 ufs_bin="/usr/bin/unionfs"
 
-ufs_options="-o cow,allow_other,direct_io,nonempty,auto_cache,sync_read"
+ufs_options="-o cow,allow_other,direct_io,nonempty,auto_cache,sync_read,uid=$uid,gid=$gid"
```

Comment:
Should uid and gid also be added to Plexdrive and Rclone?

Reply:
I don't think so, because root will have access either way. I didn't do anything special besides the above and it works fine. My grasp of this is that root will have access to the rclone mounts and then ufs will throw it over to the /local-media which will give it the other userid permission. Since ufs would still be run by root, I think that's why it works.